### ISSUE TITLE
Failed to connect to proxy

### ISSUE THREAD

**Issue by slammpete** [oxen-io/session-desktop#2712]
OK so when i send a message the app is usually stable but whenever i get a message the app freezes for at least 4-5 minutes. 

Here is my OS and other relative info . If you need more info let me know.

I have auto updates enabled and am using version session-desktop-linux-x86_64-1.10.7.AppImage.

System:
Kernel: 5.10.174-1-MANJARO arch: x86_64 bits: 64 compiler: gcc v: 12.2.1
parameters: BOOT_IMAGE=/boot/vmlinuz-5.10-x86_64
root=UUID=539dbef7-7f11-4635-8850-1e0e07764bf8 rw acpi_osi=!
"acpi_osi=Windows 2015" quiet apparmor=1 security=apparmor
udev.log_priority=3 usbcore.old_scheme_first=1
Desktop: KDE Plasma v: 5.26.5 tk: Qt v: 5.15.8 wm: kwin_x11 vt: 1 dm: SDDM
Distro: Manjaro Linux base: Arch Linux

Someone has made a package in the aur but the version is out of date.  I tried it both with and without my vpn. Also I have no trouble using tor browser or whonix on my machine. Here are some logs from your app.

{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"Handle dataMessage about convo [REDACTED] from user: [REDACTED]","time":"2023-03-19T00:33:25.223Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"Received duplicate message. Dropping it.","time":"2023-03-19T00:33:25.224Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] about to fetchReleaseFromFSAndUpdateMain","time":"2023-03-19T00:33:55.046Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] fetched latest release from fileserver:  v1.10.7","time":"2023-03-19T00:33:55.759Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] about to fetchReleaseFromFSAndUpdateMain","time":"2023-03-19T00:34:55.046Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] fetched latest release from fileserver:  v1.10.7","time":"2023-03-19T00:34:56.655Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] about to fetchReleaseFromFSAndUpdateMain","time":"2023-03-19T00:35:55.046Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] fetched latest release from fileserver:  v1.10.7","time":"2023-03-19T00:35:55.792Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] checkForUpdates","time":"2023-03-19T00:36:10.689Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] canUpdate true","time":"2023-03-19T00:36:10.690Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] checkForUpdates...","time":"2023-03-19T00:36:10.690Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] latestVersionFromFsFromRenderer v1.10.7","time":"2023-03-19T00:36:10.690Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] checkForUpdates isMoreRecent false","time":"2023-03-19T00:36:10.690Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"Fileserver has no update so we are not looking for an update from github current:1.10.7 fromFileServer:v1.10.7","time":"2023-03-19T00:36:10.690Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] about to fetchReleaseFromFSAndUpdateMain","time":"2023-03-19T00:36:55.045Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] fetched latest release from fileserver:  v1.10.7","time":"2023-03-19T00:36:55.476Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] about to fetchReleaseFromFSAndUpdateMain","time":"2023-03-19T00:37:55.046Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] fetched latest release from fileserver:  v1.10.7","time":"2023-03-19T00:37:55.502Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] about to fetchReleaseFromFSAndUpdateMain","time":"2023-03-19T00:38:55.046Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"[updater] fetched latest release from fileserver:  v1.10.7","time":"2023-03-19T00:38:55.775Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"innerHandleSwarmContentMessage","time":"2023-03-19T00:39:48.877Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"handleSwarmDataMessage","time":"2023-03-19T00:39:48.878Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"Handle dataMessage about convo [REDACTED] from user: [REDACTED]","time":"2023-03-19T00:39:48.878Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"Starting handleMessageJob for message [REDACTED] 1679186382851, undefined in conversation [REDACTED]","time":"2023-03-19T00:39:48.880Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"No downloading attachments yet as this user is not trusted for now.","time":"2023-03-19T00:39:49.061Z","v":0}
{"name":"log","hostname":"host1","pid":3794,"level":30,"msg":"Approving request for permission 'notifications'","time":"2023-03-19T00:39:53.068Z","v":0}

(session-desktop-bin:3794): libnotify-WARNING **: 19:40:18.093: Failed to connect to proxy

(session-desktop-bin:3794): libnotify-WARNING **: 19:40:43.119: Failed to connect to proxy

**Comment by slammpete** [comment]
Also and i am not sure if this applies but i am unable to connect to any exit node on lokinet to get onto the clearnet. On my desktop I built the app from source from the aur and am not using the binary.

I don't think this matters but i do live in south america but i use a vpn to connect through florida and my ping times are about 50ms and my internet speeds are 80 down 40 up. I've also tried without a vpn with the same results.
